classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -343,21 +387,22 @@ protected function get_table_columns() {
 	 * If the current request does not have any search or if this list table does not support
 	 * that feature it will return an empty string.
 	 *
-	 * TODO:
-	 *   - Improve search doing LIKE by word rather than by phrases.
-	 *
 	 * @return string
 	 */
 	protected function get_items_query_search() {
 		global $wpdb;
 
-		if ( empty( $_GET['s'] ) || empty( $this->search_by ) ) {
+		if ( empty( $_GET['s'] ) || empty( $this->search_by ) ) { //phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			return '';
 		}
 
-		$filter  = array();
+		$search_string = $wpdb->esc_like( sanitize_text_field( wp_unslash( $_GET['s'] ) ) ); //phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		$filter = array();
 		foreach ( $this->search_by as $column ) {
-			$filter[] = $wpdb->prepare('`' . $column . '` like "%%s%"', $wpdb->esc_like( $_GET['s'] ));
+			$wild     = '%';
+			$sql_like = $wild . $wpdb->esc_like( $search_string ) . $wild;
```

Comment:
Do we need to pass `$search_string` through `esc_like()` twice (looking up at like 399)?

Reply:
Of course not, somehow I've overseen this. I've left the esc_like in the SQL statement as it makes more sense there and removed the duplicate above.